crates/re_renderer/src/size.rs

```diff
@@ -12,46 +11,43 @@
 pub struct Size(pub f32);
 
 impl Size {
-    /// Automatically sized, based on a view builder setting.
-    pub const AUTO: Self = Self(f32::MAX);
+    /// Zero radius.
+    pub const ZERO: Self = Self(0.0);
 
-    /// Like [`Size::AUTO`], but larger by some small factor (~2).
-    pub const AUTO_LARGE: Self = Self(f32::MIN);
+    /// Radius of length 1 in scene units.
+    pub const ONE: Self = Self(1.0);
```

Comment:
```suggestion
    pub const ONE_SCENE: Self = Self(1.0);
```

…though is this even used?

Reply:
rename shows it's not used. Let's just remove it.
I didn't call it _SCENE so far because it's technically SCENE_UNIT plus everything ways "default is scene" so might as well.